ui/components/requirements/requirement.jsx

```diff
@@ -7,6 +7,7 @@ export default function Requirement({ requirement }) {
     <span key={idx} className="req-text-line mb1">{ line }<br /></span>
   ));
   /* eslint-enable react/no-array-index-key */
+  const sunsetString = requirement.policy.sunset ? 'Sunset date by $requirement.policy.sunset' : 'Sunset date: none';
```

Comment:
I think you'd like to use string interpolation here:
```javascript
`Sunset date by ${requirement.policy.sunset}`
```

It'd be worthwhile to add a unit test here -- hopefully the examples in `ui/__test__` explain what do to, but we can dive in deeper if you'd like.